src/Kiota.Builder/Refiners/PhpRefiner.cs

```diff
@@ -107,5 +148,33 @@ private static void CorrectParameterType(CodeElement codeElement)
             });
             CrawlTree(codeElement, CorrectParameterType);
         }
+
+        private static void AliasUsingWithSameSymbol(CodeElement currentElement) {
+            if(currentElement is CodeClass {StartBlock: CodeClass.Declaration currentDeclaration} currentClass && currentDeclaration.Usings.Any(x => !x.IsExternal)) {
+                var duplicatedSymbolsUsings = currentDeclaration.Usings
+                    .Distinct(usingComparer)
+                    .GroupBy(x => x.Declaration.Name, StringComparer.OrdinalIgnoreCase)
+                    .Where(x => x.Count() > 1)
+                    .SelectMany(x => x)
+                    .Union(currentDeclaration
+                        .Usings
+                        .Where(x => !x.IsExternal)
+                        .Where(x => x.Declaration
+                            .Name
+                            .Equals(currentClass.Name, StringComparison.OrdinalIgnoreCase)));
+                var rand = new Random();
```

Comment:
we can't use random, to subsequent generation will result in different code, making it painful to review weekly generations.
Look at what I've done for Go, you might even be able to reuse as is.